custom/icds_reports/templates/icds_reports/mobile_dashboard_download.html

```diff
@@ -0,0 +1,134 @@
+{% extends "hqwebapp/base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% block title %}{% trans "ICDS Dashboard Mobile Application" %}{% endblock %}
+{% block tabs %}{% endblock tabs %}
+{% block content %}
+  <section class="section">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-3">
+          <img style="width: 100%" src="{% static 'icds/img/ICDS-CAS_banner.png' %}" alt="ICDS-CAS"/>
+        </div>
+        <div class="col-sm-9">
+          <div>
+            <h2>Download the ICDS-CAS Mobile Dashboard Application</h2>
+            <a class="btn btn-primary" href="{% static 'media/cas-mobile-dashboard.apk' %}">
+            <i class="fa fa-download"></i> Get the App
+          </a>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+          <div>
+            <h2>Complete File List</h2>
+            <table class="table">
+              <thead>
+              <tr>
+                <th>File</th>
+                <th>Purpose</th>
+                <th>Version Number</th>
+                <th>Last Updated</th>
+              </tr>
+              </thead>
+              <tbody>
+              <tr>
+                <td><a href="{% static 'media/cas-mobile-dashboard.apk' %}">cas-mobile-dashboard.apk</a></td>
+                <td>Setup file to install CAS Mobile Dashboard on your device</td>
+                <td>1.0</td>
+                <td>28th January 2020</td>
+              </tr>
+              <tr>
+                <td><a href="#">Installation Instructions</a></td>
```

Comment:
Should this link go somewhere? Same for the overview below.

Reply:
Yeah it will, still working on getting the files (the link to this page is behind a flag)